HelpSource/Classes/Server.schelp

```diff
@@ -504,6 +504,25 @@ Plot the node/group tree. As link::#-queryAllNodes:: but graphical.
 argument:: interval
 Polling interval.
 
+
+method:: serverBooting
+A method that aims at returning a boolean stating wether server is booting or not
```

Comment:
Don't write "A method that aims at returning" – that is really too complicated! :) Compare: https://github.com/supercollider/supercollider/blob/master/HelpSource/Classes/Thread.schelp#L163